examples/dev_flow/debug/debugging_visual.rst

```diff
@@ -0,0 +1,72 @@
+
+.. _examples_dev_flow_debug_visual:
+
+
+Debugging shared libraries with Visual Studio
+=============================================
+
+Conan packages don't contain the necessary information for debugging libraries with Visual Studio by default.
+This information is stored in PDBs that are generated during the compilation of the libraries. When using Conan these
+PDBs are generated in the build folder, which isn't always available, making impossible to debug the dependencies.
+
```

Comment:
I would say something like "...which may not be available..." and explain that the reason why is because you have cleaned you cache of the builds using `conan cache clean` for example...